guides/plugins/plugins/framework/message-queue/add-message-handler.md

```diff
@@ -153,19 +146,21 @@ For more information on this check the [Symfony docs](https://symfony.com/doc/cu
 
 ### Transport
 
-A [transport](https://symfony.com/doc/current/messenger.html#transports-async-queued-messages) is responsible for communicating with your 3rd party message broker. You can configure multiple transports and route messages to multiple or different transports. Supported are all transports that are either supported by [Symfony](https://symfony.com/doc/current/messenger.html#transport-configuration) itself, or by [Enqueue](https://github.com/php-enqueue/enqueue-dev/tree/master/docs/transport). If you don't configure a transport, messages will be processed synchronously like in the Symfony event system.
+A [transport](https://symfony.com/doc/current/messenger.html#transports-async-queued-messages) is responsible for communicating with your 3rd party message broker. You can configure multiple transports and route messages to multiple or different transports. Supported are all transports that are either supported by [Symfony](https://symfony.com/doc/current/messenger.html#transport-configuration) itself. If you don't configure a transport, messages will be processed synchronously like in the Symfony event system.
 
-You can configure an amqp transport directly in your `framework.yaml`, but since Shopware is integrated with Enqueue, it is best practice to configure your transport in your `enqueue.yaml` and simply tell Symfony to use your enqueue transports.
+You can configure an amqp transport directly in your `framework.yaml` and simply tell Symfony to use your  transports.
 
-In your `enqueue.yaml` you simply configure your transports as follows:
+In your `framework.yaml` you simply configure your transports as follows:
 
 {% code title="<platform root>/src/Core/Framework/Resources/config/packages/enqueue.yaml" %}
 
 ```yaml
-enqueue:
-  default: # the name of your transport
-    transport: ~ # the transport configuration
-    client: ~ # the client configuration
+framework:
+  messenger:
+    transports:
+      default: # the name of your transport
```

Comment:
Before it was relatable to see that the CLI command `bin/console messenger:consume default --memory-limit=128M` refers to this config `enqueue.default`. Now you changed the CLI command to `bin/console messenger:consume async --memory-limit=128M` but kept the configuration example to name the transport default. Was this on purpose or a missing change?

Reply:
It's a missing change